engines/director/detection_tables.h

```diff
@@ -2070,11 +2075,15 @@ static const DirectorGameDescription gameDescriptions[] = {
 	// Original filename is ピーターと狼
 	// One gameplay mode supports General MIDI
 	MACDEMO1_l("mipeterwolf", "Demo", "Peter and the Wolf", "f5277c53bacd27936158dd3867e587e2", 394909, Common::JA_JPN, 311),
+	MACDEMO1_l("mipeterwolf", "Demo", "xn--o9juj9b8hb1942h", "f5277c53bacd27936158dd3867e587e2", 394644, Common::JA_JPN, 311),
+	// Decomposed variant of the above filename
```

Comment:
For the decomposed variants that you add, can you explain the scenario where someone's filenames would decompose? Could this happen when copying files between platforms?

Reply:
On Mac OS, Unicode filenames on the filesystem are canonically in decomposed format. Filesystem APIs that fetch/interact with filenames receive decomposed strings. ScummVM doesn't do Unicode normalization before comparing strings, so having only the composed form in the detection tables means that ScummVM will fail to identify the files on Mac OS.